recipes-kernel/linux/files/iot2050_defconfig_base

```diff
@@ -684,10 +653,10 @@ CONFIG_CMA_SIZE_MBYTES=24
 CONFIG_PRINTK_TIME=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_DEBUG_FS=y
+# CONFIG_SLUB_DEBUG is not set
 CONFIG_SCHEDSTATS=y
 # CONFIG_DEBUG_PREEMPT is not set
 # CONFIG_FTRACE is not set
 CONFIG_SAMPLES=y
 CONFIG_SAMPLE_RPMSG_CLIENT=m
 CONFIG_MEMTEST=y
-CONFIG_TI_DAVINCI_MDIO=y
```

Comment:
There are a lot of changes to this file - how did you come to them?

Reply:
Let's do it like this:
 - switch to built-in prueth in this commit
 - drop 5.10 recipes
 - regenerate the base defconfig in a separate commit on top

Reply:
Since the base config was from 5.10 I first ran olddefconfig and then menuconfig to enable PRUETH, but I can drop the olddefconfig step and let the build process do it by itself